feature-libs/checkout/b2b/root/checkout-b2b-root.module.ts

```diff
@@ -31,14 +26,9 @@ export const CHECKOUT_B2B_CMS_COMPONENTS: string[] = [
 export function defaultCheckoutComponentsConfig() {
   const config: CmsConfig = {
     featureModules: {
-      [CHECKOUT_B2B_FEATURE]: {
+      [CHECKOUT_FEATURE]: {
```

Comment:
just putting out there since we're doing this all in one name

then it's fine if we don't align with cart-lib when they did CART_BASE_FEATURE
All in all, it wouldn't make sense anymore with this new setup

Reply:
Yep. We have other checkouts than just the base one, so it wouldn't make sense to use the base in the name